.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
           - black == 22.3.0
           - usort == 1.0.2
 
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/pycqa/flake8
```

Comment:
@pmeier  I'm hitting a login wall on the gitlab url. Do we need to patch main?

Reply:
See #6953. This branch is just stale.